src/src/targets.h

```diff
@@ -95,32 +97,47 @@
 Credit to Jacob Walser (jaxxzer) for the pinout!!!
 https://github.com/jaxxzer
 */
-#define GPIO_PIN_NSS            PB12
-#define GPIO_PIN_BUSY           -1 // NOT USED ON THIS TARGET 
-#define GPIO_PIN_DIO0           PA15
-#define GPIO_PIN_DIO1           PA1 // NOT CORRECT!!! PIN STILL NEEDS TO BE FOUND BUT IS CURRENTLY UNUSED
-#define GPIO_PIN_MOSI           PB15
-#define GPIO_PIN_MISO           PB14
-#define GPIO_PIN_SCK            PB13
-#define GPIO_PIN_RST            PC14
+#define GPIO_PIN_NSS            PB12 //confirmed on SLIMPLUS
+#define GPIO_PIN_BUSY           -1   // NOT USED ON THIS TARGET
+#define GPIO_PIN_DIO0           PA15 //confirmed on SLIMPLUS
+#define GPIO_PIN_DIO1           PA1  // NOT CORRECT!!! PIN STILL NEEDS TO BE FOUND BUT IS CURRENTLY UNUSED
+/////////////////////////////////////// NOT FOUND ON SLIMPLUS EITHER.
+#define GPIO_PIN_MOSI           PB15 //confirmed on SLIMPLUS
+#define GPIO_PIN_MISO           PB14 //confirmed on SLIMPLUS
+#define GPIO_PIN_SCK            PB13 //confirmed on SLIMPLUS
+#define GPIO_PIN_RST            PC14 //confirmed on SLIMPLUS
 #define GPIO_PIN_SDA            PB7
 #define GPIO_PIN_SCL            PB6
+
 #ifdef USE_R9MM_R9MINI_SBUS
     #define GPIO_PIN_RCSIGNAL_RX    PA3
     #define GPIO_PIN_RCSIGNAL_TX    PA2
-#else
+#elif TARGET_R9SLIMPLUS_RX               // R9SLIMPLUS USES DUAL UART CONFIGURATION FOR TX1/RX1
+    #define GPIO_PIN_RCSIGNAL_RX    PB11 // RX1 PIN OF CONNECTOR 1 ON SLIMPLUS
+    #define GPIO_PIN_RCSIGNAL_TX    PA9  // TX1 PIN OF CONNECTOR 1 ON SLIMPLUS
+#else //default R9MM_R9MINI or R9MX
     #define GPIO_PIN_RCSIGNAL_RX    PA10
     #define GPIO_PIN_RCSIGNAL_TX    PA9
 #endif
+
 #ifdef TARGET_R9MX_RX
     #define GPIO_PIN_LED            PB2 // Red
     #define GPIO_PIN_LED_RED        PB2 // Red
-    #define GPIO_PIN_LED_GREEN      PB3 // Green 
+    #define GPIO_PIN_LED_GREEN      PB3 // Green
     #define GPIO_PIN_BUTTON         PB0  // pullup e.g. LOW when pressed
-#else
+#elif TARGET_R9SLIMPLUS_RX
+    #define GPIO_PIN_LED            PA11 // Red
+    #define GPIO_PIN_LED_RED        PA11 // Red
+    #define GPIO_PIN_LED_GREEN      PA12 // Green
+    #define GPIO_PIN_BUTTON         PC13  // pullup e.g. LOW when pressed
+    /* PB3: RX = HIGH, TX = LOW */
+    #define GPIO_PIN_RX_ENABLE PB3
+    /* PB9: antenna 1 (left) = HIGH, antenna 2 (right) = LOW
```

Comment:
This should not be left floating and be set to a single antenna.

Reply:
Ok :)

Reply:
fixed